azurerm/validators.go

```diff
@@ -76,6 +77,118 @@ func validateIso8601Duration() schema.SchemaValidateFunc {
 	}
 }
 
+func validateAzureVirtualMachineTimeZone() schema.SchemaValidateFunc {
+	// Candidates are listed here: http://jackstromberg.com/2017/01/list-of-time-zones-consumed-by-azure/
+	return validation.StringInSlice([]string{
```

Comment:
appears this comes from the .net method `TimeZoneInfo.FindSystemTimeZoneById()` - with this appearing to be a [source from Microsoft for the information](https://support.microsoft.com/en-us/help/973627/microsoft-time-zone-index-values)?

Reply:
According to my test this is not the full list. For example, Azure also accepts "UTC-02" which is not in the list. And the weird thing is, Azure rejects "UTC-03".

Reply:
honestly that feels like a bug in the API - as if I saw `UTC-02` on the list, as an operator I'd assume that `UTC-3` or `UTC-03` would work?